crates/services/gas_price_service/src/fuel_gas_price_updater/fuel_da_source_adapter/dummy_source.rs

```diff
@@ -0,0 +1,14 @@
+use crate::fuel_gas_price_updater::{
+    service::Result as DaGasPriceSourceResult,
+    DaGasPrice,
+    DaGasPriceSource,
+};
+
+pub struct DummyDaGasPriceSource;
```

Comment:
Maybe `DummyDaBlockCosts` would be better.

Reply:
addressed in https://github.com/FuelLabs/fuel-core/pull/2163/commits/74fc9bf4e08cb0d4a1a5def23f31d2c4fbd31c9f#diff-3985ff8ef4e8622f1508b6665a9bf061b8a02fed5c53b1abfaecf072e88ceaa2R7